rust/remotivelabs-broker/examples/configuration/interfaces.json

```diff
@@ -0,0 +1,23 @@
+{
+  "default_namespace": "VirtualInterface",
+  "chains": [
+    {
+      "type": "virtual",
+      "device_name": "virtual",
+      "namespace": "VirtualInterface"
+    }
+  ],
+  "gateway": {
```

Comment:
can we simplify this configuration? ie remove things that are not used?

Reply:
Removed unused parameters.